packages/service-library/src/servicelib/rabbitmq/_utils.py

```diff
@@ -51,7 +53,13 @@ async def wait_till_rabbitmq_responsive(url: str) -> bool:
 
 
 def get_rabbitmq_client_unique_name(base_name: str) -> str:
-    return f"{base_name}_{socket.gethostname()}"
+    # NOTE: below prefix is guaranteed to change each time the preocess restarts
```

Comment:
MINOR: some typos. Here is rewritten by chatty
```
# NOTE: The prefix below will change every time the process restarts.
# Why is this necessary?
# 1. The codebase relies on this behavior; without it, subscribers and consumers will fail.
# 2. It allows the web server to be restarted seamlessly during [re]deployments.
```

Reply:
We are keeping the typos bere and I'll put it in another PR